src/CMakeLists.txt

```diff
@@ -76,8 +76,7 @@ else()
 endif()
 
 
-set(EXTRALIBS uuid rt dl
-  ${Boost_LIBRARIES} ${Boost_SYSTEM_LIBRARY} ${ATOMIC_OPS_LIBRARIES})
+set(EXTRALIBS uuid rt dl ${Boost_LIBS} ${ATOMIC_OPS_LIBRARIES})
```

Comment:
@tchaikov ${Boost_LIBS} neither in /usr/share/cmake/modules/FindBoost.cmake nor anywhere else in the source tree. It should be changed to ${Boost_LIBRARIES}